packages/apps/graph/src/graph/query/blocks-from-height.ts

```diff
@@ -27,6 +33,12 @@ builder.queryField('blocksFromHeight', (t) =>
         },
       }),
     },
+    validate(args) {
+      if (args.endHeight && args.startHeight > args.endHeight) {
+        return false;
+      }
+      return true;
+    },
```

Comment:
Do we return an proper error message in the response when the value is incorrect?

Also this could be simplified:

```suggestion
    validate: args => !(args.endHeight && args.startHeight > args.endHeight)
```

Reply:
So instead of validating it here, and throwing an error if it fails, I just passed this validation to the resolver and I threw an input error with the correct custom message

Reply:
Let me know if it looks good